src/Tools/dotnet-monitor/LoggingEventIds.cs

```diff
@@ -88,7 +88,8 @@ internal enum LoggingEventIds
         SharedLibraryPath = 75,
         ConnectionModeConnect = 76,
         ConnectionModeListen = 77,
-        ExperimentalFeatureEnabled = 78
+        ExperimentalFeatureEnabled = 78,
+        StartCollectingArtifact = 79
```

Comment:
nit: this appears to be `StartCollectArtifact` everywhere else; consider unifying naming